pkgs/tools/package-management/harmonia/default.nix

```diff
@@ -30,6 +30,13 @@ rustPlatform.buildRustPackage rec {
     nix
   ];
 
+  # not compatible with current nix version:
+  # $ harmonia
+  # The program must call nix::initNix() before calling any libstore library functions.
+  # [1]    148943 IOT instruction (core dumped)  ./result/bin/harmonia
+  # See https://github.com/nix-community/harmonia/ for a maintained fork
```

Comment:
Shouldn't this be packaged, then?

Reply:
Why not use your fork?

Reply:
I had asked the people behind the project already what to do with. Response was like that:

> I don't know 100% how to deal with this. Because actually i want to keep supporting harmonia but i'm really busy right now. i'm sorry we haven't upstreamed harmonia into the nixpkgs either, the package in our repo is still building. just against nix2.11 iirc

So they are still using the project but they do not see maintaining a nixpkgs package as their priority. I didn't wanted to steal their name from them...

Reply:
I would also upstream the nixos module if we would switch to it...

Reply:
I set up a vote on what we should do here: https://github.com/NixOS/nixpkgs/issues/221498

Reply:
Given this, I aprove breaking this with some message.